plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestDynamicTable.java

```diff
@@ -183,7 +183,9 @@ public void testFilterWithPushdownConstraint()
     public void testFilterWithUdf()
```

Comment:
Can you please let me know what are you fixing here? Is this a flaky issue? Do you have any github issues reported for the problem you are fixing? How the problem you are fixing manifests?